client/my-sites/upgrades/checkout/supporting-text.jsx

```diff
@@ -6,6 +6,7 @@ var React = require( 'react' );
  * Internal dependencies
  */
 var cartValues = require( 'lib/cart-values' ),
+	support = require( 'lib/support' ),
```

Comment:
The pattern we've used with paths in most places that I've seen are to include them in the same directory as the components in a file named `paths.js`.

All the files updated here live in `my-sites/upgrades`, which already contains a `paths` module - is there a reason you didn't include these there?